contacts/app.run.js

```diff
@@ -0,0 +1,37 @@
+import ContactsService from './contacts.service';
+import {
+  reRenderContactsListHTML,
+  reRenderFavouritesListHTML
+} from './html-helper';
+import {
+  eventsHandlerContacts
+} from './contacts-handle';
+import {
+  deleteAllButtonAction,
+  clearButtonAction,
+  addNewPerson,
+  changeAddButtonState
+} from './buttons.helper';
+
+
+const main = () => {
+
+  reRenderFavouritesListHTML();
+
+  reRenderContactsListHTML();
+
+};
+
+
+main();
+
+
+window.ui = {
+  eventsHandlerContacts,
```

Comment:
All this functions should be defined in the main file, because they are events handlers.